crates/wasi-common/wig/src/lib.rs

```diff
@@ -38,6 +38,11 @@ pub fn define_wasi_struct(args: TokenStream) -> TokenStream {
     wasi::define_struct(args.into()).into()
 }
 
+#[proc_macro]
+pub fn define_wasi_struct_for_wiggle(args: TokenStream) -> TokenStream {
+    wasi::define_struct_for_wiggle(args.into()).into()
+}
```

Comment:
Is it temporary to add this hook in `wig` here?

Reply:
Ah, I missed the part of your earlier comment where you explained this ^_^.

Reply:
Yep, until we figure out where to put the glue macro for `wasmtime-wasi`/`wasmtime`, i.e., the one that's autogenerating the `Func` wrappers, and whatnot.